src/current/v23.2/take-and-restore-locality-aware-backups.md

```diff
@@ -28,7 +28,7 @@ For a technical overview of how a locality-aware backup works, refer to [Job coo
 Locality-aware backups are available in **CockroachDB {{ site.data.products.dedicated }}**, **CockroachDB {{ site.data.products.serverless }}**, and **CockroachDB {{ site.data.products.core }}** clusters when you are running [customer-owned backups](https://www.cockroachlabs.com/docs/cockroachcloud/take-and-restore-customer-owned-backups). For a full list of features on CockroachDB Cloud, refer to [Backup and Restore Overview](https://www.cockroachlabs.com/docs/cockroachcloud/backup-and-restore-overview).
```

Comment:
Should these dedicated and serverless variables be updated?

Reply:
Yes! But I'll do it in my next PR as it is part of self-hosted nav section.